tests/installation/ipxe_install.pm

```diff
@@ -156,27 +201,24 @@ sub run {
 
     select_console 'sol', await_console => 0;
 
-    #use ipxe bootloader in O3
-    #it is static menu and choose the TW entry to start installation
-    if (get_var('VIRT_AUTOTEST') && is_tumbleweed) {
-        save_screenshot;
-        ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
-        assert_screen('o3-ipxe-menu', 300);
-        my $key = get_var('HOST_INSTALL_AUTOYAST') ? (is_kvm_host ? 'k' : 'z') : 't';
-        send_key "$key";
-        # try one more time as sometimes sending key does not take effect
-        send_key "$key" if check_screen('o3-ipxe-menu');
+    # Print screenshots for ipxe boot process
+    if (get_var('VIRT_AUTOTEST') || get_var('HANA_PERF')) {
+        #it is static menu and choose the TW entry to start installation
+        enter_o3_ipxe_boot_entry if is_tumbleweed and get_var('OPENQA_URL') =~ "openqa.opensuse.org";
         assert_screen([qw(load-linux-kernel load-initrd)], 240);
         # Loading initrd spend much time(fg. 10-15 minutes to Beijing SUT)
         # Downloading from O3 became much more quick, some needles may not be caught.
-        check_screen([qw(start-tw-install network-config-created loading-installation-system)], 360);
-        if (match_has_tag("start-tw-install")) {
-            record_info("TW install", "Start with installing Tumbleweed...");
+        check_screen([qw(start-tw-install start-sle-install)], 120);
+        if (match_has_tag('start-tw-install')) {
+            record_info("Install TW", "Start installing Tumbleweed...");
+        }
+        elsif (match_has_tag('start-sle-install')) {
+            record_info("Install SLE", "Start installing SLE...");
         }
         else {
-            record_info("TW install?", "Pls make sure it is Tumbleweed that is being installed.", result => 'softfail');
+            record_info("Install others?", "Pls make sure the product that is expected to be installed.");
         }
```

Comment:
Do we need to add the  needles per sles product? If not, that's fine. Otherwise maybe a little expensive just for record_info.

Reply:
I have added needles for TW and SLE seperately, no need to add needles for each SLE release.